helpers/voting/formatVotes.ts

```diff
@@ -90,14 +96,15 @@ export function formatVotesToReveal(decryptedVotesForUser: VoteT[]) {
   return decryptedVotesForUser.flatMap((vote) => {
     if (vote.isRevealed || !vote.isCommitted || !vote.decryptedVote) return [];
 
-    const { identifier, decryptedVote, ancillaryData, time } = vote;
+    const { identifier, decryptedVote, ancillaryData, ancillaryDataL1, time } =
+      vote;
     const { price, salt } = decryptedVote;
 
     return {
       identifier,
       time,
       price,
-      ancillaryData,
+      ancillaryData: ancillaryDataL1 ?? ancillaryData,
```

Comment:
this is one part im not sure about, when we hash the vote, do we use the L1 data?

Reply:
Yes!